File: fix_dash.py
import io

with io.open(r'app\dashboard\page.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

old = "{ label: '\U0001F4CA Reporting', href: '/reporting', activites: ['ck_dress'] },"
new = old + "\n  { label: 'Gestionnaire Stock', href: '/gestionnaire-stock', activites: ['ck_dress', 'ck_design', 'succes_design'] },"

if 'gestionnaire-stock' not in content:
    content = content.replace(old, new)

with io.open(r'app\dashboard\page.tsx', 'w', encoding='utf-8', newline='') as f:
    f.write(content)

print("OK")

File: fix_dash.js
const fs = require('fs');
const path = 'app/dashboard/page.tsx';
let content = fs.readFileSync(path, 'utf8');

const old = "{ label: '📊 Reporting', href: '/reporting', activites: ['ck_dress'] },";
const add = old + "\n  { label: 'Gestionnaire Stock', href: '/gestionnaire-stock', activites: ['ck_dress', 'ck_design', 'succes_design'] },";

if (!content.includes('gestionnaire-stock')) {
  content = content.replace(old, add);
}

fs.writeFileSync(path, content, 'utf8');
console.log('OK - lien ajoute sans BOM');